track/templates/en/guidance.html

```diff
@@ -3,13 +3,73 @@
 {% block title %}Read guidance{% endblock %}
 {% block pageid_en %}guidance{% endblock %}
 {% block pageid_fr %}directives{% endblock %}
-{% block description %}Read guidance.{% endblock description %}
+{% block description %}Project guidance.{% endblock description %}
 
 {% block content %}
 <section id="main-content" class="flex-1">
 	<div class="container mx-auto items-center sm:w-4/5 xl:w-3/5 mt-6 mb-6">
-		<h1>Read guidance</h1>
-		<p>Page content here</p>
+		<h1>Project guidance</h1>
+		
+        <ol>
```

Comment:
I didn't realize that tailwind doesn't apply the same styles to lists as paragraphs by default, odd! If you change <ol> to <ol class="text-lg mb-2 leading-normal"> the style will match the rest of the website

Reply:
tx Eva will do this